Packages/Store/Sources/Stores/NodeStore.swift

```diff
@@ -73,7 +73,15 @@ public struct NodeStore {
                 .deleteAll(db)
         }
     }
-    
+
+    public func deleteNode(url: String, chain: String) throws {
```

Comment:
```suggestion
    public func deleteNode(chain: String, url: String) throws {
```

chain is higher in hierarchy